compiler/q-implant/README.md

```diff
@@ -1,14 +1,67 @@
 # q-implant
 
-_q-implant_ writes quantization parameters and weights (given as .json and .npy files) to a circle model.
+_q-implant_ implants quantization parameters and weights 
+(given as .json and .npy files) to a circle model.
+
+## Format of input files (`.json` and `.npy`)
+
+The main input file is a `.json` file, which is a dictionary.
+
+The dictionary's key is a tensor name, and its value is quantization parameters
+and values(if exists).
+
+The `.npy` file is a NumPy file that is generated through the `numpy.save` function. 
```

Comment:
.npy can be generated with numpy.save or other functions. So I slightly rephrased the sentence.

```suggestion
The `.npy` file is a NumPy file that can be generated with `numpy.save` function. 
```